content/messaging/customer-chat-api/v3.3/rtm-reference/index.mdx

```diff
@@ -3340,6 +3341,7 @@ Informs about an incoming [greeting](https://www.livechatinc.com/help/why-should
   "unique_id": "Q10O0N5B5D",
   "displayed_first_time": true,
   "addon": "email",
+  "subtype": "announcement",
```

Comment:
The last update, I promise :) I would also add a short description of this field to the table "Push payload".

subtype | Currently, announcement is the only greeting subtype.